http-testkit-client/src/test/scala/com/wix/e2e/http/client/internals/BUILD.bazel

```diff
@@ -0,0 +1,10 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
delete this file, no bazel build here

Reply:
Wow, did bazel idea plugin just create it?